Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,39 @@
 import React from 'react';
 import './App.css';
+import ActionButton from './components/ButtonComponents/ActionButton';
+import NumberButton from './components/ButtonComponents/NumberButton';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className="calculator">
+      <CalculatorDisplay />
+      <div>
+        <ActionButton text="clear" />
+        <NumberButton buttonStyle="red" text="/" />
+      </div>
```

Comment:
this set up is verrrrry similar to the solution, not that you couldnt have come to it on your own, but if you did, do beware leaning on the solution during project time, try not to look until after